exercism/hamming/hamming.spec.js

```diff
@@ -7,31 +7,31 @@ describe('Hamming', function () {
     expect(hamming.compute('A', 'A')).toEqual(0);
```

Comment:
Unsure of the @@ code before this expect function in your test. Perhaps the describe() was supposed to be on the next line?